font-test-data/src/ift.rs

```diff
@@ -3,6 +3,8 @@
 //! Used for incremental font transfer. Specification:
 //! <https://w3c.github.io/IFT/Overview.html>
 
+use std::iter::repeat;
```

Comment:
Totally absolute nit: I avoid importing free functions directly, and instead import the containing module and then call the functions through the module, because it makes it easier to understand what is being called. (e.g. I'd `use std::iter;` and then call `iter::repeat(0)`).

This used to actually be in the official style guide but I don't see it anymore, so really only make this change if you actually think it makes sense.